app/jobs/leader_check_ins_job.rb

```diff
@@ -7,13 +7,12 @@ class LeaderCheckInsJob < ApplicationJob
   LEADER_PIPELINE_KEY = Rails.application.secrets.streak_leader_pipeline_key
 
   def perform(usernames = [])
-    usernames = active_leader_usernames if usernames.empty?
+    user_ids = active_leadre_slack_ids
 
-    usernames.each do |username|
-      user = user_from_username(username)
-      next if user.nil? # couldn't find the user
+    user_ids = user_ids_from_usernames usernames unless usernames.empty
```

Comment:
No, this statement checks whether it has been provided a list of usernames – if it has it only starts check ins with those, as opposed to every active leader.